samples/python/django/models.py

```diff
@@ -0,0 +1,76 @@
+''' Copyright 2022 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+'''
+
+from django.db import models
+from django.db.models import Q, F
+class Singer(models.Model):
+  class Meta():
+    db_table = 'singers'
+  id = models.IntegerField(primary_key=True, null=False)
```

Comment:
Is it possible to move the properties that are common to all models into some kind of an abstract base model. That would then include the properties `id`, `createdAt` and `updatedAt`.

Reply:
Moved `created_at` and `updated_at` to the base mode but since we want `id` to be primary key, it needs to be in each of the models itself.